pipelines/utilities/kubeconfig.sh

```diff
@@ -12,6 +12,8 @@ set_kubeconfig(){
   elif [[ "${TF_VAR_arch}" == "arm64"  ]]; then
     export KUBECONFIG="${PWD}/test_framework/terraform/${LONGHORN_TEST_CLOUDPROVIDER}/${DISTRO}/k3s.yaml"
   fi
+
+  cat "${KUBECONFIG}"
```

Comment:
Could this expose credentials outside in the job build?

Reply:
From my understanding there are 3 fields in KUBECONFIG: `certificate-authority-data`, `client-certificate-data` and `client-key-data`. `certificate-authority-data` is SSL server certificate. `client-certificate-data` and `client-key-data` are also `openssl` generated certificate and key which to be approved to access the k8s cluster, so they don't involve personal ssh key or credential. Even though one gets this KUBECONFIG file, he can only manipulate the created k8s cluster / ec2 instances, and has nothing to do with our infra machines.

Reply:
We don't directly print out `kubeconfig` now, but users can ssh into control plane node to retrieve it. For example, the default file path for `k3s` is `/etc/rancher/k3s/k3s.yaml`.